pkg/names/names.go

```diff
@@ -24,8 +24,14 @@ import (
 )
 
 const (
-	ServerContainer      = "server"
-	ServerContainerIndex = 0
+	ServerContainer = "server"
+	NMAContainer    = "nma"
+	Index0          = 0
```

Comment:
Can we rename this to `FirstContainer`? I think this is only used when wanted to reference index [0]